bootstrap/apollo-env.sh

```diff
@@ -14,3 +16,10 @@ if [[ -z "${color_start-}" ]]; then
   declare -r color_norm="${color_start}0m"
 fi
 
```

Comment:
I dont personally think we need to make it as complicated as a level thing (3 -vvv would be fine) - but i'm happy to be put in my place if you have a good argument for it.